WinFormsUI/MainForm.cs

```diff
@@ -1,4 +1,5 @@
 ﻿using System;
+using System.Collections.Generic;
```

Comment:
Codacy found an issue: [Remove this unnecessary 'using'.](https://app.codacy.com/manual/siriak/MoneyManager/pullRequest?prid=6066087)